com.obsproject.Studio.json

```diff
@@ -0,0 +1,157 @@
+{
+  "app-id": "com.obsproject.Studio",
+  "branch": "stable",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.9",
+  "sdk": "org.kde.Sdk",
+  "command": "obs",
+  "rename-icon": "obs",
+  "rename-desktop-file": "obs.desktop",
+  "finish-args": [
+    "--socket=x11",
```

Comment:
KDE recommendation is now also to include wayland as I remember.

Reply:
If you use Wayland OBS is less than useful considering it can't capture anything.

Reply:
Fair point